sdk/resourcemanager/automanage/armautomanage/examples.go

```diff
@@ -0,0 +1,159 @@
+//go:build go1.18
```

Comment:
For this kind of sample code, we have a separated [repo](https://github.com/Azure-Samples/azure-sdk-for-go-samples/tree/main/sdk/resourcemanager) to maintain. Could you please migrate this example to that repo?

Reply:
Removed.